src/components/Modal/Modal.vue

```diff
@@ -16,26 +16,25 @@
         <header
           v-if="hasHeader"
           id="modalTitle"
-          class="flex relative justify-between border-b border-gray-100 pb-4"
+          class="flex justify-between border-b border-gray-100 pb-4"
         >
           <slot name="header" />
-          <close
+          <button
+            :class="['rounded-full w-7 h-7 p-1 cursor-pointer hover:bg-white-200',
+                     'focus:outline-none focus:ring-2 focus:ring-primary-100']"
             aria-label="Close modal"
```

Comment:
Probably out of scope for this change but we really should be passing in aria-label values rather than hard coding them in the component.  Not only might we want the label to be something different we want it to be translated as well.  I'll create a ticket for it.

Reply:
https://lobsters.atlassian.net/browse/DANG-1422